### DISCUSSION TITLE
self.model in the trainer class

### DISCUSSION THREAD

**Discussion by georgemilosh** [georgemilosh/Climate-Learning#59] [Ideas]
It is a bit annoying to do development currently since. Wouldn't it be convenient @AlessandroLovo , if trainer.run method returned also the model it was training?

**Comment by AlessandroLovo** [comment]
`trainer.run` performs k fold cross validation, hence 10 models are involved

**Reply by georgemilosh** [reply]
Yes I understand that. However there will be a benefit in having one of them, at least, to run some tests. Also I think it would be convenient to be able to work with a single fold
